docs/config/lua/keyassignment/SetWindowLevel.md

```diff
@@ -0,0 +1,28 @@
+# `SetWindowLevel`
+
+Set window level sepcified by the argument value. eg: `AlwaysOnTop` keeps the current window on top of other windows.
+
+Accepted values: `AlwaysOnBottom` | `Normal` | `AlwaysOnTop`
+
+```lua
+local act = wezterm.action
+local config = {}
+
+config.keys = {
+    {
+        key = '[',
+        mods = 'CMD',
+        action = act.SetWindowLevel("AlwaysOnBottom")
+    }, 
+    {
+        key = '0',
+        mods = 'CMD|SHIFT',
+        action = act.SetWindowLevel("Normal")
+    }, 
+    {
+        key = ']',
+        mods = 'CMD',
+        action = act.SetWindowLevel("AlwaysOnTop")
+    }, 
+}
```

Comment:
this feels a bit more self-contained and focused on the assignment that is being discussed in this doc page

```suggestion
config.keys = {
    {
        key = '[',
        mods = 'CMD',
        action = wezterm.action.SetWindowLevel("AlwaysOnBottom")
    }, 
    {
        key = '0',
        mods = 'CMD|SHIFT',
        action = wezterm.action.SetWindowLevel("Normal")
    }, 
    {
        key = ']',
        mods = 'CMD',
        action = wezterm.action.SetWindowLevel("AlwaysOnTop")
    }, 
}
```